portfolio/src/main/webapp/scripts/servlet.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Calls to Datastore servlet to load and add comments
+ */
+
+
+/** Takes the max number of comments available to the user from datastore */
+function setMaxNumberOfComments() {
+  fetch('/comments-size').then(response => response.text()).then((commentArraySize)  => {
+    const maxArraySize = parseInt(commentArraySize);
+    const maxNumberofComments = document.getElementById('number-of-comments');
+
+    maxNumberofComments.max = maxArraySize.toString(); // Set the max number as attribute of the input
+  });
+}
+
+/** Takes the number of comments the user wants to see and displays them */
+function loadComments() {
+  const numberOfComments = document.getElementById('number-of-comments').value;
+  fetch(`/load-comments?number-of-comments=${numberOfComments}`)
+    .then(response => response.json()).then((comments) => {
+    setMaxNumberOfComments(); // Update the maximum number of comments available.
+    appendComments(comments);
+  });
+}
+
+/** Eliminate displayed comments to load a new set of them */
+function removePassedComments(commentWrapper) {
```

Comment:
```suggestion
function removeCommentsFromDOM(commentWrapper) {
```
Nit: I might be a bit more specific with this function name, but this is just personal preference.